src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -22,20 +23,34 @@ import { Toolbar } from "./Toolbar";
 import { clearEditor } from "./utils/editor/clear";
 import { parseRawValue } from "./utils/editor/parseRawContent";
 
+export type TextStylesType = {
```

Comment:
I'd maybe rename this to just `TextStyles` because it includes not only the types but also the classNames